### ISSUE TITLE
Difference in parameters between config.yaml and paper

### ISSUE THREAD

**Issue by Merlin-721** [jjery2243542/adaptive_voice_conversion#34]
## Config.yaml vs Paper
* lambda_kl:  1 vs 0.01
* batch_size:  128 vs 256
* dropout_rate:  0 vs 0.5 on all layers

Anyone know the reason for this? Are these updated values that should be used?

Most notably the batch_size would affect the number of training iterations, where the current 200k would only perform half the training with 128 samples per batch, vs 256 samples per batch.

**Comment by jjery2243542** [comment]
Hi, I found that some hyperparameters (ex. dropout) are unnecessary so I updated the number in this repo. Feel free to tune the number.
